crates/utils/re_byte_size/src/arrow_sizes.rs

```diff
@@ -13,17 +13,24 @@ impl SizeBytes for dyn Array {
     }
 }
 
-impl SizeBytes for ArrayRef {
+impl SizeBytes for &dyn Array {
```

Comment:
This should be removed, `impl SizeBytes for dyn Array` already covers it since the methods take `&self`.